helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -277,10 +277,7 @@ private Response batchGetStoppableInstances(String clusterId, JsonNode node, boo
           .error(String.format("Current cluster %s has issue with health checks!", clusterId), e);
       throw new HelixHealthException(e);
     } catch (Exception e) {
-      _logger.error(String.format(
-              "Failed to get parallel stoppable instances for cluster %s with a HelixException!",
-              clusterId), e);
-      throw e;
+      throw new Exception("Failed to get parallel stoppable instances for cluster " + clusterId + " with a HelixException!", e);
```

Comment:
probably we do not want to change the exception time as it may change caller's behavior if they have a catch for certain exception. If no variable is logged, can we just remove the catch block.

Reply:
Not sure I understand. Is throwing a new Exception with `e` passed in as arguments not the same time as exception `e` by itself?

Reply:
The caught exception is a base type, the actual object can be any derived type (for example, HelixException). The object you throw is an object of the base type. 
If there is a try-catch block in caller that catch any derived type (for example, HelixException), the base type exception won't be caught. -> we will have behavior change.